tinydancer-noop/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```

Comment:
It was just the "auto generated" file that came with the new module. There is nothing in it.